Update to PHP 7.2 Causes Failure(s) in DrupalVm

My system information:
* BLT version: 9.2.3

When updating to PHP 7.2 in my box/config.yml file, I got a failed task related to XDebug. 

When I run this command:
```
vagrant reload --provision
```

I get the following output:
```
TASK [geerlingguy.php-xdebug : Build Xdebug.] **********************************
changed: [<project>] => (item=phpize)
failed: [<project>] (item=./configure) => {"changed": true, "cmd": ["./configure"], "delta": "0:00:00.929691", "end": "2019-01-04 16:41:26.711354", "item": "./configure", "msg": "non-zero return code", "rc": 1, "start": "2019-01-04 16:41:25.781663", "stderr": "configure: WARNING: You will need re2c 0.13.4 or later if you want to regenerate PHP parsers.\nconfigure: error: not supported. Need a PHP version >= 5.5.0 and < 7.2.0 (found 7.2.13-1+ubuntu16.04.1+deb.sury.org+1)", "stderr_lines": ["configure: WARNING: You will need re2c 0.13.4 or later if you want to regenerate PHP parsers.", "configure: error: not supported. Need a PHP version >= 5.5.0 and < 7.2.0 (found 7.2.13-1+ubuntu16.04.1+deb.sury.org+1)"], "stdout": "checking for grep that handles long lines and -e... /bin/grep\nchecking for egrep... /bin/grep -E\nchecking for a sed that does not truncate output... /bin/sed\nchecking for cc... cc\nchecking whether the C compiler works... yes\nchecking for C compiler default output file name... a.out\nchecking for suffix of executables... \nchecking whether we are cross compiling... no\nchecking for suffix of object files... o\nchecking whether we are using the GNU C compiler... yes\nchecking whether cc accepts -g... yes\nchecking for cc option to accept ISO C89... none needed\nchecking how to run the C preprocessor... cc -E\nchecking for icc... no\nchecking for suncc... no\nchecking whether cc understands -c and -o together... yes\nchecking for system library directory... lib\nchecking if compiler supports -R... no\nchecking if compiler supports -Wl,-rpath,... yes\nchecking build system type... x86_64-pc-linux-gnu\nchecking host system type... x86_64-pc-linux-gnu\nchecking target system type... x86_64-pc-linux-gnu\nchecking for PHP prefix... /usr\nchecking for PHP includes... -I/usr/include/php/20170718 -I/usr/include/php/20170718/main -I/usr/include/php/20170718/TSRM -I/usr/include/php/20170718/Zend -I/usr/include/php/20170718/ext -I/usr/include/php/20170718/ext/date/lib\nchecking for PHP extension directory... /usr/lib/php/20170718\nchecking for PHP installed headers prefix... /usr/include/php/20170718\nchecking if debug is enabled... no\nchecking if zts is enabled... no\nchecking for re2c... no\nchecking for gawk... no\nchecking for nawk... nawk\nchecking if nawk is broken... no\nchecking whether to enable Xdebug support... yes, shared\nchecking Check for supported PHP versions... ", "stdout_lines": ["checking for grep that handles long lines and -e... /bin/grep", "checking for egrep... /bin/grep -E", "checking for a sed that does not truncate output... /bin/sed", "checking for cc... cc", "checking whether the C compiler works... yes", "checking for C compiler default output file name... a.out", "checking for suffix of executables... ", "checking whether we are cross compiling... no", "checking for suffix of object files... o", "checking whether we are using the GNU C compiler... yes", "checking whether cc accepts -g... yes", "checking for cc option to accept ISO C89... none needed", "checking how to run the C preprocessor... cc -E", "checking for icc... no", "checking for suncc... no", "checking whether cc understands -c and -o together... yes", "checking for system library directory... lib", "checking if compiler supports -R... no", "checking if compiler supports -Wl,-rpath,... yes", "checking build system type... x86_64-pc-linux-gnu", "checking host system type... x86_64-pc-linux-gnu", "checking target system type... x86_64-pc-linux-gnu", "checking for PHP prefix... /usr", "checking for PHP includes... -I/usr/include/php/20170718 -I/usr/include/php/20170718/main -I/usr/include/php/20170718/TSRM -I/usr/include/php/20170718/Zend -I/usr/include/php/20170718/ext -I/usr/include/php/20170718/ext/date/lib", "checking for PHP extension directory... /usr/lib/php/20170718", "checking for PHP installed headers prefix... /usr/include/php/20170718", "checking if debug is enabled... no", "checking if zts is enabled... no", "checking for re2c... no", "checking for gawk... no", "checking for nawk... nawk", "checking if nawk is broken... no", "checking whether to enable Xdebug support... yes, shared", "checking Check for supported PHP versions... "]}
failed: [<project>] (item=make) => {"changed": true, "cmd": ["make"], "delta": "0:00:00.007886", "end": "2019-01-04 16:41:29.047282", "item": "make", "msg": "non-zero return code", "rc": 2, "start": "2019-01-04 16:41:29.039396", "stderr": "make: *** No targets specified and no makefile found.  Stop.", "stderr_lines": ["make: *** No targets specified and no makefile found.  Stop."], "stdout": "", "stdout_lines": []}

```

And I expected this to happen:
the vm to reprovision

**Solution**
This is directly related to - https://github.com/geerlingguy/ansible-role-php-xdebug/issues/55 and requires you to NOT just update PHP in the VM but also XDebug.

--- a/scripts/drupal-vm/config.yml
+++ b/scripts/drupal-vm/config.yml
@@ -71,7 +71,7 @@ drupal_install_site: false
 configure_drush_aliases: false
 
 # This is required for front-end building tools.
-nodejs_version: "6.x"
+nodejs_version: "8.x"
 nodejs_npm_global_packages:
   - name: bower
   - name: gulp-cli
@@ -88,7 +88,7 @@ php_packages_extra:
   - imagemagick
 
 # XDebug configuration.
-php_xdebug_version: 2.5.0
+php_xdebug_version: 2.6.1
 # Change this value to 1 in order to enable xdebug by default.
 php_xdebug_default_enable: 0
 php_xdebug_coverage_enable: 0

